src/System.Private.ServiceModel/src/System/ServiceModel/Channels/StreamedFramingRequestChannel.cs

```diff
@@ -0,0 +1,368 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime;
+using System.Security.Authentication.ExtendedProtection;
+using System.ServiceModel;
+using System.ServiceModel.Channels.ConnectionHelpers;
+using System.ServiceModel.Security;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ServiceModel.Channels
+{
+    internal class StreamedFramingRequestChannel : RequestChannel
+    {
+        internal IConnectionInitiator connectionInitiator;
+        internal ConnectionPool connectionPool;
```

Comment:
{AncestorPath = 54131590}
Also `this.connectionInitator` and `this.connectionPool` below needs to be changed